MantidPlot/make_package.rb.in

```diff
@@ -210,13 +210,18 @@ if( "@MAKE_VATES@" == "ON" )
   `install_name_tool -id @rpath/libNonOrthogonalSource.dylib plugins/paraview/qt4/libNonOrthogonalSource.dylib`
 end
 
-# use install_tool_name to add an @rpath for the libraries
+# use install_tool_name to add an @rpath for the python dynamic libraries
 Dir["Contents/MacOS/*.so"].each do |library|
-  dependencies = `otool -L #{library}`
   print library, "\n"
   `install_name_tool -add_rpath @loader_path/../MacOS #{library} > /dev/null 2>&1`
 end
 
+# use install_tool_name to add an @rpath for the ParaView python dynamic libraries
```

Comment:
I was wondering if this is MantidPlot only, or the script for the workbench should also be tweaked?

Reply:
In this case this is only required for MantidPlot as we are not shipping ParaView/Vates with workbench.

For the move to High Sierra I'm getting together a single script that will cover both so we don't have this concern going forward.

Reply:
Seems there is a problem with `readline.py` when packaging, was there a recent change regarding this?

Reply:
This is a problem on the macOS 10.13 server as that no longer exists. I've got a separate branch with fixes for building on 10.13 but we annoyingly can't force PR builds onto 10.10.

Reply:
`NDW1170` was free so I've started a new build and it's building on OSX 10.10 now - https://builds.mantidproject.org/job/pull_requests-osx/31826/